.github/workflows/build.yml

```diff
@@ -10,4 +10,14 @@ jobs:
           ./config_linux.sh
           make
           cd picsimlab
+          make
+  build-macos:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: |
+          ./configure
```

Comment:
config_linux.sh only pass flag -fPIC to code to be compiled as dynamic library.
`./configure CFLAGS=-fPIC CXXFLAGS=-fPIC`
This is necessary on macOS ?

Reply:
I don't think so. Take a look at this build log on macOS https://github.com/hsorbo/uCsim_picsimlab/runs/5962440464?check_suite_focus=true#step:3:2713

the .dylib is created